applications/GeoMechanicsApplication/tests/cpp_tests/test_thermal_dispersion_law.cpp

```diff
@@ -0,0 +1,115 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Anne van de Graaf
+//                   Gennady Markelov
+//
+
+#include "custom_constitutive/thermal_dispersion_law.hpp"
+#include "geo_mechanics_application.h"
+#include "includes/ublas_interface.h"
+#include "testing/testing.h"
+
+namespace Kratos::Testing {
+
+KRATOS_TEST_CASE_IN_SUITE(CalculateThermalDispersionMatrix2D, KratosGeoMechanicsFastSuite)
+{
+    Model current_model;
+    auto& r_model_part = current_model.CreateModelPart("ModelPart", 1);
```

Comment:
The default value of the buffer size is 1, so we can leave it out here. (For this test case, that size doesn't matter, I think.)
```suggestion
    auto& r_model_part = current_model.CreateModelPart("ModelPart");
```

Reply:
1 is removed